products/knownprojects/dbt_project.yml

```diff
@@ -7,3 +7,6 @@ model-paths: ["sql"]
 tests:
   +store_failures: true
   schema: "_tests"
+
+vars:
+  max_duplicate_projects: 250
```

Comment:
just curious, does this variable represent unique count of records that have dupes or the total count of records? For example, I have 2 unique id records; each id has 4 more dupes. So the total is 10 records with 2 unique ids...

Reply:
I believe it's unique count of records that have dupes

and I think the variable name should be changed to reflect it's used to test "project records" rather than "projects". I'll change that

Reply:
I shouldn't put a screenshot of the data here but an example of the outputs is

| unique_field | n_records |
|--------|--------|
| a_123 | 2 |
| b_123 | 50 |
| c_123 | 3 | 

and this would contribute 3 toward that threshold of 250 duplicates. a recent run of this [here](https://github.com/NYCPlanning/data-engineering/actions/runs/7835217216/job/21380150443#step:10:45) showed 65 and the relevant table has 65 rows like those above

Reply:
Nice!!! Thanks for the example!